cambridgeScript/parser/lexer.py

```diff
@@ -89,15 +108,13 @@ def __eq__(self, other):
             return True
         return super().__eq__(other)
 
-
-# Regex patterns for tokens
 _TOKENS = [
     ("IGNORE", r"/\*.*\*/|(?://|#).*$|[ \t]+"),
     ("NEWLINE", r"\n"),
     ("KEYWORD", "|".join(Keyword)),
+    ("SYMBOL", r"-|(" + "|".join(re.escape(s) for s in Symbol if s != "-") + ")"),
     ("LITERAL", r'-?[0-9]+(?:\.[0-9]+)?|".*?"'),
```

Comment:
Using `re.escape` is fine but reordering is most certainly not (I suspect that's why you broke minus sign parsing).

```suggestion
    ("LITERAL", r'-?[0-9]+(?:\.[0-9]+)?|".*?"'),
    ("SYMBOL", r"(" + "|".join(map(re.escape, Symbol) + ")"),
```

Reply:
applied suggested change, but a minus sign followed by a literal (like `a-1`) will still be recognized as `-1`.  `a - 1` is correctly parsed

Reply:
Oh my bad, I didn't notice that parsing issue; that probably means unary/binary minus resolution has to happen in the parser, but fixing that in a separate PR would be better.